### ISSUE TITLE
`withDefaultVariant` is not exported from "@chakra-ui/react" in v2.9.3

### ISSUE THREAD

**Issue by jakeboone02** [chakra-ui/chakra-ui#8880]
### Description

The v2.9.3 [documentation for `withDefaultVariant`](https://v2.chakra-ui.com/docs/styled-system/customize-theme#theme-extension-withdefaultvariant) has this import statement:

```ts
import { extendTheme, withDefaultVariant } from '@chakra-ui/react';
```

But `withDefaultVariant` is not exported from "@chakra-ui/react" in v2.9.3. I did this instead:

```ts
import { extendTheme } from "@chakra-ui/react";
import { withDefaultVariant } from "@chakra-ui/theme-utils";
```

### Link to Reproduction

https://codesandbox.io/p/sandbox/crtmvf?file=%2Ftheme.ts%3A3%2C1-4%2C1

### Chakra UI Version

2.9.3

### Browser

Build issue...browser is irrelevant

### Operating System

- [X] macOS
- [X] Windows
- [X] Linux

**Comment by segunadebayo** [comment]
This is now fixed in 2.9.4. Thanks for letting us know.

**Comment by jakeboone02** [comment]
Confirmed. Thanks for the quick response!
